Winapp2.ini

```diff
@@ -122,544 +198,941 @@ FileKey10=%ProgramFiles%\Google\Update\Offline|*.*|RECURSE
+Detect4=HKCU\Software\CentBrowser
+Detect5=HKCU\Software\ChromePlus
+Detect6=HKCU\Software\Chromium\PreferenceMACs
+Detect7=HKCU\Software\CocCoc\Browser
+Detect8=HKCU\Software\Comodo\Dragon
+Detect9=HKCU\Software\Coowon\Coowon
+Detect10=HKCU\Software\Epic Privacy Browser
+Detect11=HKCU\Software\Flock
+Detect12=HKCU\Software\RockMelt
+Detect13=HKCU\Software\Slimjet
+Detect14=HKCU\Software\Torch
+Detect15=HKCU\Software\Vivaldi
+Detect16=HKCU\Software\Yandex\YandexBrowser
+DetectFile1=%AppData%\brave
+DetectFile2=%LocalAppData%\Google\Chrome*
+Default=False
+FileKey1=%AppData%\brave|QuotaManager
+FileKey2=%LocalAppData%\7Star\7Star\User Data\*|QuotaManager
+FileKey3=%LocalAppData%\360Browser\Browser\User Data\*|QuotaManager
+FileKey4=%LocalAppData%\Amigo\User Data\*|QuotaManager
+FileKey5=%LocalAppData%\CentBrowser\User Data\*|QuotaManager
+FileKey6=%LocalAppData%\Chromium\User Data\*|QuotaManager
+FileKey7=%LocalAppData%\CocCoc\Browser\User Data\*|QuotaManager
+FileKey8=%LocalAppData%\Comodo\Dragon\User Data\*|QuotaManager
+FileKey9=%LocalAppData%\Coowon\Coowon\User Data\*|QuotaManager
+FileKey10=%LocalAppData%\Epic Privacy Browser\User Data\*|QuotaManager
+FileKey11=%LocalAppData%\Flock\User Data\*|QuotaManager
+FileKey12=%LocalAppData%\Google\Chrome*\User Data\*|QuotaManager
+FileKey13=%LocalAppData%\MapleStudio\ChromePlus\User Data\*|QuotaManager
+FileKey14=%LocalAppData%\RockMelt\User Data\*|QuotaManager
+FileKey15=%LocalAppData%\Slimjet\User Data\*|QuotaManager
+FileKey16=%LocalAppData%\Torch\User Data\*|QuotaManager
+FileKey17=%LocalAppData%\Vivaldi\User Data\*|QuotaManager
+FileKey18=%LocalAppData%\Yandex\YandexBrowser\User Data\*|QuotaManager
 
 [Session Storage *]
 LangSecRef=3029
-SpecialDetect=DET_CHROME
-Detect1=HKCU\Software\7Star
-Detect2=HKCU\Software\Chromium
-Detect3=HKCU\Software\CocCoc
-Detect4=HKCU\Software\Coowon
-Detect5=HKCU\Software\SuperBird
-Detect6=HKCU\Software\Torch
-Detect7=HKCU\Software\Vivaldi
-Default=False
-FileKey1=%AppData%\Brave\*\Session Storage|*.*|RECURSE
+Detect1=HKCU\Software\7Star\7Star
+Detect2=HKCU\Software\360Browser\Browser
+Detect3=HKCU\Software\Amigo
+Detect4=HKCU\Software\CentBrowser
+Detect5=HKCU\Software\ChromePlus
+Detect6=HKCU\Software\Chromium\PreferenceMACs
+Detect7=HKCU\Software\CocCoc\Browser
+Detect8=HKCU\Software\Comodo\Dragon
+Detect9=HKCU\Software\Coowon\Coowon
+Detect10=HKCU\Software\Epic Privacy Browser
+Detect11=HKCU\Software\Flock
+Detect12=HKCU\Software\RockMelt
+Detect13=HKCU\Software\Slimjet
+Detect14=HKCU\Software\Torch
+Detect15=HKCU\Software\Vivaldi
+Detect16=HKCU\Software\Yandex\YandexBrowser
+DetectFile1=%AppData%\brave
+DetectFile2=%LocalAppData%\Google\Chrome*
+Default=False
+FileKey1=%AppData%\brave\Session Storage|*.*|RECURSE
 FileKey2=%LocalAppData%\7Star\7Star\User Data\*\Session Storage|*.*|RECURSE
-FileKey3=%LocalAppData%\Amigo\User Data\*\Session Storage|*.*|RECURSE
-FileKey4=%LocalAppData%\Chrome Plus\User Data\*\Session Storage|*.*|RECURSE
-F
```

Comment:
Can this be changed to \Mozilla\Firefox? This path exists on my machine with Nightly installed but \Mozilla\Mozilla Firefox does not 

Alternatively, we can add HKLM\SOFTWARE\Mozilla\Nightly as a detect

Reply:
Unfortunately, the registry key "HKLM\\SOFTWARE\\Mozilla\\Firefox" also often exists on systems without a Firefox installation. Therefore, I changed the Detect to DetectFile, because this will cover all Firefox versions with just one line.